### PR TITLE
[PM-3198][PM-3199] Rename PasswordlessLoginStrategy

### PR DESCRIPTION
**Pull request by rr-bw**
## Type of change

- Tech debt (refactoring, code cleanup, dependency upgrades, etc)

## Objective

- Rename `PasswordlessLoginStrategy` to `AuthRequestLoginStrategy`
- Rename `LoginWithDeviceComponent` to `LoginViaAuthRequestComponent`

**Additional Renaming**
- This PR also standardizes the naming of our login strategies to use `Login` instead of `LogIn`
   - `AuthRequestLoginCredentials`
   - `PasswordLoginCredentials`
   - `UserApiLoginCredentials`
   - `SsoLoginCredentials`

### CODE REVIEW

File: apps/web/src/app/oss-routing.module.ts [hunk 1]
```diff
@@ -67,12 +67,12 @@ const routes: Routes = [
       { path: "login", component: LoginComponent, canActivate: [UnauthGuard] },
       {
         path: "login-with-device",
-        component: LoginWithDeviceComponent,
+        component: LoginViaAuthRequestComponent,
         data: { titleId: "loginWithDevice" },
```
Comment (rr-bw) [inline apps/web/src/app/oss-routing.module.ts:71]: @JaredSnider-Bitwarden Do we need any changes to `path` or `data` properties? See also the Browser and Desktop `app-routing.module.ts` files

Comment (JaredSnider-Bitwarden) [reply apps/web/src/app/oss-routing.module.ts:71]: The `titleId` shouldn't change. It's a translation id used in the `header.component`

Comment (JaredSnider-Bitwarden) [reply apps/web/src/app/oss-routing.module.ts:71]: Hmm... I missed setting a separate translation for this web route:
```   
{
        path: "admin-approval-requested",
        component: LoginWithDeviceComponent,
        data: { titleId: "loginWithDevice" },
  },
```

File: libs/angular/src/auth/components/login-via-auth-request.component.ts [hunk 2]
```diff
@@ -32,15 +32,13 @@ import { PasswordGenerationServiceAbstraction } from "@bitwarden/common/tools/ge
...
 
-// TODO: consider renaming this component something like LoginViaAuthReqComponent
-
 enum State {
   StandardAuthRequest,
   AdminAuthRequest,
 }
 
 @Directive()
-export class LoginWithDeviceComponent
```
Comment (JaredSnider-Bitwarden) [inline libs/angular/src/auth/components/login-via-auth-request.component.ts:43]: This file has a lot of remaining references to `Passwordless` items we could clean up as well.

**Review by JaredSnider-Bitwarden — DISMISSED** [review]
Thank you for all the name refactors!

**Review by jlf0dev — DISMISSED** [review]
LGTM 👍

**Review by JaredSnider-Bitwarden — APPROVED** [review]
LGTM!

**Review by jlf0dev — APPROVED** [review]

**Pull request merged**